### PR TITLE
Issue #7: show upload/download more specific progress

### PR DESCRIPTION
**Pull request by carlosolimpio**
While uploading a file the progress was only showing for huge files

- Defined CHUNK_SIZE constant to hold the default file's chunk size
- Set it to each download/upload function
- Updated each print function to show a more detailed progress

### CODE REVIEW

File: gdrive [hunk 1]
```diff
@@ -12,6 +12,8 @@ import sys
 import mimetypes
 from os.path import expanduser
 
+CHUNK_SIZE = 1024*256
```
Comment (viniciusalbuquerque) [inline gdrive:15]: Maybe we should discuss to define how CHUNK_SIZE can affect our code.

File: gdrive [hunk 2]
```diff
@@ -75,18 +77,18 @@ def upload(service, filepath):
...
                                         media_body=media,
                                         fields='id')
         response = None
-        print("Uploading 0%", end='\r')
+        print("Uploading 0.0%", end='\r')
         while response is None:
             status, response = created_file.next_chunk()
             if status:
-                print("Uploading %d%%" % int(status.progress() * 100),end='\r')
+                print("Uploading %.1f%%" % float(status.progress() * 100), end='\r')
```
Comment (viniciusalbuquerque) [inline gdrive:89]: I think I would prefer to have this as it was and having another information that would show the amount that has already being download. Something like `50% 50MB/100MB`. This second information could be updated more frequently. What do you think?

File: gdrive (hunk 1 above)
Comment (carlosolimpio) [reply gdrive]: After a test with download function I decided to leave as 5MB since the default size was 100MB.

File: gdrive [hunk 3]
```diff
@@ -63,6 +65,12 @@ def get_filename(service, file_id):
     print('Filename is "%s" for ID: %s.' % (filename, file_id))
     return filename
 
+def sizeof_fmt(num, suffix='B'):
```
Comment (CarlosRodrigo) [inline gdrive:68]: what's fmt? please rename to size_of_**full_meaning**

Comment (carlosolimpio) [reply gdrive]: renamed to size_to_human_readable

**Pull request merged**